lib/kennel/models/slo.rb

```diff
@@ -19,6 +19,11 @@ class Slo < Record
         monitor_ids: -> { DEFAULTS.fetch(:monitor_ids) }
       )
 
+      def initialize(project, *args)
+        super(project, *args)
+        raise ValidationError, 'Threshold warning must be less-than critical value' if thresholds.each { |t| t[:warning].to_f < t[:critical].to_f }
```

Comment:
`.each` is always true ;)

... need `.any?`
... or `thresholds.each {  raise "foo #{warning} #{critical}" if bar`